examples/samplers/simulated_annealing_sampler.py

```diff
@@ -5,7 +5,7 @@
 
 Note that this implementation isn't intended to be used for production purposes and
 has the following limitations:
-- The sampler only supports `UniformDistribution` (i.e., `Trial.suggest_uniform` method).
+- The sampler only supports `UniformDistribution` (i.e., `Trial.suggest_float` method).
```

Comment:
This line should not be changed.

Reply:
I agree.  `Trial.suggest_float` corresponds to `LogUniformDistribution` in addition to `UniformDistribution`, and it may be confusing for readers. An alternative fix will be to add condition of the `log` argument, for example, as follows:

```
i.e., `Trial.suggest_float` method with `log=False`
or
i.e., `Trial.suggest_float` method without the `log` option
```

Reply:
Okay, leave this line as it was :slightly_smiling_face: